cloud/deployment/status.go

```diff
@@ -16,26 +16,50 @@ import (
 const (
 	// ErrInvalidStatus represents an invalid deployment status error.
 	ErrInvalidStatus errors.Kind = "invalid deployment status"
+
+	// ErrInvalidFilterStatus represents an invalid deployment status filter error.
+	ErrInvalidFilterStatus errors.Kind = "invalid filter stack status"
```

Comment:
Intended or forgotten to change?
```suggestion
	ErrInvalidFilterStatus errors.Kind = "invalid filter deployment status"
```

Reply:
so much copy paste, sorry.